README.md

```diff
@@ -7,7 +7,7 @@
 ### Installation
 
    - install library
-     - `pip install https://github.com/genestack/python-client/archive/stable.zip --upgrade`
+     - `pip install genestack-client --upgrade`
```

Comment:
One doesn't need `--upgrade` to install the package, I suggest we remove it for clarity.